app/models/tag.rb

```diff
@@ -7,4 +7,8 @@ class Tag < ApplicationRecord
   validates_format_of :value, # because of to_param overriding.
                       :with => /\A[a-zA-Z_][a-zA-Z0-9_\- ()]*\z/,
                       :message => "should begin with an alphabet or underscore, and only contain alphanumeric letters, underscore, hyphen, space, or round brackets!"
+
+  def used_in_entries?
+    EntryTag.where(tag_id: self.id).exists?
```

Comment:
```suggestion
    entry_tags.exists?
```
でも行けそうな気がします。
あまり自信はありません。

Reply:
ありがとうございます。提案頂いた修正でも同じ返しになったので、[記述修正](https://github.com/pubannotation/pubdictionaries/pull/71/commits/da39781f8439a939e35a203440775e1381cde690)にて修正しました。